docs/examples/101/azure-search-create/main.bicep

```diff
@@ -0,0 +1,60 @@
+param name string {
+  minLength: 2
+  maxLength: 60
+}
+
+param sku string {
+  default: 'standard'
+  allowed: [
+    'free'
+    'basic'
+    'standard'
+    'standard2'
+    'standard3'
+    'storage_optimized_l1'
+    'storage_optimized_l2'
+  ]
+}
+
+param replicaCount int {
+  default: 1
+  minValue: 1
+  maxValue: 12
+}
+
+param partitionCount int {
+  default: 1
+  allowed: [
+    1
+    2
+    3
+    4
+    6
+    12
+  ]
+}
+
+param hostingMode string {
+  default: 'default'
+  allowed: [
+    'default'
+    'highDensity'
+  ]
+}
+
+param location string {
```

Comment:
change this to

```
param location string = resourceGroup().location
```

Reply:
I just made the same pattern on all pull requests I made. Is there a reason to privilegiate this pattern over the one I used?